assets/src/edit-story/components/dropTargets/provider.js

```diff
@@ -128,25 +130,30 @@ function DropTargetsProvider({ children }) {
           elementId: activeDropTargetId,
           properties: { resource, type: resource.type },
         });
+
+        // Reset styles on visisble elements
+        (currentPage?.elements || [])
+          .filter(
+            ({ id }) => !(id in Object.keys(dropTargets)) && id !== selfId
+          )
+          .forEach((el) => {
+            pushTransform(el.id, {
+              dropTargets: { active: false, replacement: null },
+            });
+          });
+        setActiveDropTargetId(null);
+
         if (selfId) {
-          deleteElementById({ elementId: selfId });
+          deleteSelectedElements();
```

Comment:
The previous version kind of felt a bit safer. Does this work correctly for both media panel and canvas dnd?

Reply:
This is temporary until https://github.com/daybrush/moveable/issues/197 is resolved, I'll add a comment. It works fine with media panel because `selfId` would be `null`